examples/Example1.cpp

```diff
@@ -24,11 +24,11 @@
 //
 //
 //  This piece of code was developed as part of ICE-DIP project at CERN.
-//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's 
+//  "ICE-DIP is a European Industrial Doctorate project funded by the European Community's
 //  7th Framework programme Marie Curie Actions under grant PITN-GA-2012-316596".
 //
 
-#include "../UMESimd.h"
```

Comment:
I'm doing what I can with documentation. If I had enough time to write more, I would write it, rest assured. So far code of the interface and scalar emulation IS effectively the documentation, and for that reason it should be in plain sight. Maybe in a while we could consider moving these files somewhere else, but for now I'd prefer it to stay where it is.

You are not taking psychological aspect into consideration. What are the first things that you do when looking at new piece of code? For me these are README file and the source code: examples + main headers. One problem I have with some projects is that there is no proper structuring between the code that is the actual definition of what the code does, and how it does it. In UME::SIMD the intention was, from the beginning, to make sure this distinction is clear and readable.